src/physics/thermal_conduction.cpp

```diff
@@ -106,7 +118,24 @@ void ThermalConduction::completeSetup()
   if (timestepper_ != serac::TimestepMethod::QuasiStatic) {
     // If dynamic, assemble the mass matrix
     M_form_ = temperature_.createOnSpace<mfem::ParBilinearForm>();
-    M_form_->AddDomainIntegrator(new mfem::MassIntegrator());
+
+    // If both a specific heat capacity and density have been specified, use their product as the mass matrix
+    // coefficient
+    if (cp_ && rho_) {
+      mass_coef_ = std::make_unique<mfem::ProductCoefficient>(*rho_, *cp_);
+    }
+    // otherwise, just use the one that is specified
+    else if (cp_) {
+      mass_coef_ = std::move(cp_);
+    } else if (rho_) {
+      mass_coef_ = std::move(rho_);
+    }
+    // if neither is specified, use the default coefficient of 1.0
+    else {
+      mass_coef_ = std::make_unique<mfem::ConstantCoefficient>(1.0);
+    }
```

Comment:
is this equivalent to putting
```cpp
cp_ = std::make_unique<mfem::ConstantCoefficient>(1.0);
rho_ = std::make_unique<mfem::ConstantCoefficient>(1.0);
```
somewhere in the constructor?

Reply:
....that's a much better way to do this. I'll go change it!